crates/astria-sequencer/README.md

```diff
@@ -69,10 +83,66 @@ I[2023-05-16|16:53:56.786] service start    module=abci-client
 ```sh
 # initialize the node
 cometbft init
-# set the block time to 15s
-sed -i'.bak' 's/timeout_commit = "1s"/timeout_commit = "15s"/g' ~/.cometbft/config/config.toml
+# set the block time to 14s
+sed -i'.bak' 's/timeout_commit = "0s"/timeout_commit = "15s"/g' ~/.cometbft/config/config.toml
 # start the node
 cometbft node
 ```
 
 You should see blocks being produced.
+
+You can also use `just` to run the above commands:
+
+```bash
+just run-cometbft
+```
+
+## Testnet
```

Comment:
I kinda liked having the testnet instructions in a different file, maybe just add a link to it here instead?